### PR TITLE
added more e2e test cases

### PR DESCRIPTION
**Pull request by AlexB3939**
added more e2e test cases

## Test plan

n/a

### CODE REVIEW

File: e2e/src/test-cases/numpy.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+addTestCase('Vector dot product', {
+    // Codebase to test against.
+    codebase: 'github.com/numpy/numpy',
+    // Context type.
+    context: 'embeddings',
+    transcript: [
+        {
+            question: 'How do I calculate the dot product of two vectors in NumPy?',
+            // Facts we want to see in the response.
+            facts: ['Dot product'],
```
[the remark this answers is not in the document]
Comment (AlexB3939) [reply e2e/src/test-cases/numpy.ts:9]: I am having trouble finding the function `literalFacts`. I searched on Github, VS Code, and [Sourcegraph code search ](https://sourcegraph.com/search?q=context:global+repo:github.com/sourcegraph/cody+literalfacts&patternType=standard&sm=1&groupBy=repo) and don't see it. Where is that function defined?

[the remark this answers is not in the document]
Comment (novoselrok) [reply e2e/src/test-cases/numpy.ts:9]: Oh, I haven't actually merged the PR that adds those functions 🤦 Ignore the comment 😄

[the remark this answers is not in the document]
Comment (AlexB3939) [reply e2e/src/test-cases/numpy.ts:9]: Okay no worries, should I just merge as is then

[the remark this answers is not in the document]
Comment (novoselrok) [reply e2e/src/test-cases/numpy.ts:9]: Almost, you should still split the fact into separate elements in this comment: https://github.com/sourcegraph/cody/pull/729/files#r1298117462

File: e2e/src/test-cases/numpy.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+addTestCase('Ufunc explanation', {
+    codebase: 'github.com/numpy/numpy',
+    context: 'embeddings',
+    transcript: [
+        {
+            question: 'Please provide a detailed explanation of ufuncs in NumPy',
+            facts: ['Ufunc'],
+            answerSummary:
+                'An explanatin of NumPy ufucs including what they are, common examples, and how they work under the hood',
```
Comment (novoselrok) [inline e2e/src/test-cases/numpy.ts:42]:
```suggestion
                'An explanation of NumPy ufuncs, including what they are, common examples, and how they work under the hood',
```

**Review by novoselrok — APPROVED** [review]

**Pull request merged**